apps/web/src/components/Dropdown/Dropdown.js

```diff
@@ -4,24 +4,41 @@ import { cn } from 'util/index'
 import { isEmpty } from 'lodash'
 import { position } from 'util/scrolling'
 import Icon from 'components/Icon'
+import { useDropdown } from 'contexts/DropdownContext'
 import classes from './Dropdown.module.scss'
 
-const Dropdown = ({ children, className, triangle, items, toggleChildren, alignRight, menuAbove, noOverflow }) => {
+const Dropdown = ({ children, className, triangle, items, toggleChildren, alignRight, menuAbove, noOverflow, id }) => {
   const [active, setActive] = useState(false)
   const parentRef = useRef(null)
+  const { activeDropdownId, openDropdown, closeAllDropdowns } = useDropdown()
```

Comment:
dang, should have mentioned that we are also trying to move away from our Dropdown to the shadcn/radix components/ui/select, which I think handles all this already. but would need to be tested more. generally we are trying to move as much as possible to the shadcn components which are tailwind wrappers around Radix components

Reply:
but for now we can roll with this